code/modules/mob/living/living.dm

```diff
@@ -2,6 +2,7 @@
 	. = ..()
 	var/datum/atom_hud/data/human/medical/advanced/medhud = huds[DATA_HUD_MEDICAL_ADVANCED]
 	medhud.add_to_hud(src)
+	faction += "\ref[src]"
```

Comment:
Could you explain why this is here?

Reply:
all mobs have themselves in their own faction list so things won't attack them that have the correct faction. It's a lot better than using hard references.